### DISCUSSION TITLE
Help with API syntax

### DISCUSSION THREAD

**Discussion by gbengaadara** [almarklein/timetagger#510] [Q&A] [answered]
I started using Timetagger about 6 months ago and really loving it. Thank you to all who contributed to this great software. I am trying to create records via the API using curl. The syntax of my command is as below
`curl -X PUT \`
`  https://localhost/timetagger/api/v2/records \`
`  -H "authtoken: TOKEN" \`
`  -H "Content-Type: application/json" \`
`  -d '[{"description": "testing", "tags": "#value"}]'`
I get the error "Got item that is not a dict with str 'key' field." after running this command.

Can anyone help me with what I am doing wrong? Thank you.

**Comment by almarklein** [comment, marked as answer]
These are the docs you need: https://timetagger.readthedocs.io/en/stable/webapi/#object-shapes

**Comment by gbengaadara** [comment]
Thank you. I got it working now
